### ISSUE TITLE
Add skeleton screen when loading

### ISSUE THREAD

**Issue by Secretmapper** [Hack-Weekly/turquoise-cobra-2#18]
![image](https://user-images.githubusercontent.com/4077143/226137695-d055e73f-d858-4c71-92be-dcc273d310c5.png)

**Comment by thenameismiggy** [comment]
I'm gonna take this issue if no one else has started working on it yet :)

**Comment by thenameismiggy** [comment]
I've implemented a basic skeleton screen that imitates the look of the ChatRoom component. Currently figuring out if it's possible to have 2 skeleton screens: one that looks like the LoginBox if there is no logged in user and the second will be the one currently implemented now.

Also, classNames as of now are basically just copied from the ChatRoom and ChannelList files. Still have to clean this up and remove the unneeded classNames.

EDIT: I just realized that there are libraries to implement skeleton screens. Lol
